schema-engine/sql-migration-tests/tests/migrations/cockroachdb.rs

```diff
@@ -27,8 +27,8 @@ fn db_push_on_cockroach_db_with_postgres_provider_works(api: TestApi) {
         api.connection_string()
     );
 
-    let connector = migration_core::migration_api(Some(schema.clone()), None).unwrap();
-    let output = tok(connector.schema_push(migration_core::json_rpc::types::SchemaPushInput {
+    let connector = schema_core::schema_api(Some(schema.clone()), None).unwrap();
+    let output = tok(connector.schema_push(schema_core::json_rpc::types::SchemaPushInput {
```

Comment:
I tried to check out what `tok` referred to and only managed to find the following:
```rs
run_with_thread_local_runtime
```
Does "tok" mean something specific here? It's unclear to me

Reply:
It runs async code in a sync context. `tok` meaning `tokio::spawn(future).join()`.